tests/test_linter_endpoint.py

```
@@ -0,0 +1,110 @@
+    if "error" in result:
+        print(f"❌ Error: {result['error']}")
+    elif "result" in result:
+        print(f"✅ Found {result['result']['summary']['total']} issues:")
+        for issue in result['result']['results']:
+            print(f"   - Line {issue['line']}: {issue['severity'].upper()} - {issue['message']}")
+            if issue['suggestion']:
+                print(f"     💡 {issue['suggestion']}")
+
+    # Test 2: Valid contract
+    print("\n2. Testing valid contract:")
+    response = requests.post(url, json={
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)